build-tools/src/main/java/org/elasticsearch/gradle/plugin/BasePluginBuildPlugin.java

```diff
@@ -63,6 +63,7 @@ public BasePluginBuildPlugin(ProviderFactory providerFactory) {
 
     @Override
     public void apply(final Project project) {
+        project.getPluginManager().apply(BasePlugin.class);
```

Comment:
Why is this required? Does the `JavaPlugin` no longer apply the `BasePlugin`?

Reply:
I thought of applying it explicitly here as we require the BasePluginExtension in this class. But seems to cause more confusion than clarification. Removed it